scripts/base/frameworks/broker/store.zeek

```diff
@@ -57,12 +57,62 @@ export {
 		SQLITE,
 	};
 
+	# Behavior when the backend is found to be corrupt or
+	# failing and there's a possibility to get back to
+	# working by wiping/deleting state. For example, a
+	# corrupted SQLite database can be deleted.
+	type FailureMode: enum {
```

Comment:
Since Zeek's enums lose their type name when referring them outside of the module (e.g. `Broker::FAIL`) I'm wondering whether these should be `SQLITE_FAIL` and `SQLITE_DELETE`? There's already `SUCCESS` and `FAILURE` via `QueryStatus`, which is a bit close in naming. These are specific to SQLite, right?

Reply:
Yes, I was in the same corner. Loosing the prefix is quite unfortunate. It doesn't need to be completely specific to SQLite, but today it is, so I'm happy to add the prefix, but will go full `SQLITE_FAILURE_MODE_FAIL`, because `DELETE` and `FAIL` are just too generic short.

Thanks!